content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+although you usually want to customize it before deploying it.
+You may, however, want to push the unmodified version to SCM
+so you have it as part of your history.
+
+[NOTE]
+====
+The *Configuration as Code* UI page shows the full pathname of the YAML file being used
+and gives a box where you can specify a different file to use.
+See the information below about how to modify the location used for JCasC YAML files.
+====
+
+The default JCasC YAML file has four sections:
+
+* `jenkins` section defines the root Jenkins object,
+with configurations that can be set with the
+*Manage Jenkins >> Configure System*
+and *Manage Jenkins >> Configure Nodes and Clouds* screens.
+
+* `tool` section defines build tools that can be set on the
+*Manage Jenkins >> Global Tool Configuration* screen.
+
+* *unclassified* section defines all other configuration,
+including configuration for installed plugins.
+
+* `credentials` section defines credentials that can be set on the
+*Manage Jenkins >> Manage Credentials* screen.
+You may want to delete this section from your YAML file;
+this is discussed beginning at 27:53 in
+link:https://www.youtube.com/watch?v=ANU7jkxbZSM[How to Install Jenkins on CentOS 7 Using Ansible and JCasC].
+
+== YAML File Syntax
+
+JCasC defines the controller configuration in a YAML file.
+YAML is a popular serialization language for configuration information,
+with a syntax that is straight-forward and easy to read but precise.
+
+Some key points about YAML syntax:
+
+* YAML files are case sensitive.
+* Indentation is very significant and specific.
+* Each item is a key/value pair.
+** The key is followed by a colon (`:`) and a space.
+** YAML converts certain strings into other types unless they are in quotes.
+*** Values such as `true`, `false`, `Yes`, and `No` are converted to Boolean values.
+*** Values such as `2` and `3.0` are converted to floating point values.
+* A value can be a list:
+** Each list item is on a separate line starting with a dash (`-`).
+** Each list item in the file must start at the same indentation.
+*** Use spaces, never tabs, for indentation.
+** Never leave a blank line in a YAML file -- things will break!
+
+See the
+link:https://yaml.org/refcard.html[YAML Reference Card]
+for more details about YAML file syntax.
+
+== Checking the *.yaml File into SCM
+
+To get the maximum benefit of JCasC, the YAML files should be stored in SCM.
+This gives you a history that you can use to trace changes that are made
+and allow you to easily roll back to an earlier version of the file if necessary.
+
+JCasC does not require that the file be stored in SCM
+and so does not enforce any rules about how you do this.
+The most common practice is to create one SCM repository
+in which you store all of your JCasC files.
+
+If you are storing your JCasC YAML files in SCM,
+you should commit the first default file that is generated,
+before you make any modifications to the file.
+
+== Modifying the JCasC File
```

Comment:
The [style guide](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments) requests sentence case the "titles" (I interpret that to mean "headings", but that is not a consistent usage throughout the documents, for sure) (optional)

```suggestion
== Modifying the JCasC file
```